ecommerce/urls.py

```diff
@@ -49,11 +49,6 @@
         BulkEnrollCouponListView.as_view(),
```

Comment:
Should we remove this view too? If we do:

1. Some associated classes in views.py would also be removed
2. Some functions in api.py would also be removed

Reply:
Yes, Seems like we can remove this View as well.